#! /bin/sh /usr/share/dpatch/dpatch-run
## 903_skip_base_ruby_check.dpatch by akira yamada <akira@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: skip host ruby check.  (this packages uses pre-generated files.)

@DPATCH@
--- a/configure.in
+++ b/configure.in
@@ -19,8 +19,6 @@ AC_ARG_WITH(baseruby,
 	[
 		BASERUBY="ruby"
 	])
-test "`RUBYOPT=- $BASERUBY -e 'p 42' 2>/dev/null`" = 42 ||
-    BASERUBY="echo executable host ruby is required.  use --with-baseruby option.; false"
 AC_SUBST(BASERUBY)
 
 AC_DEFUN([RUBY_MINGW32],
